spec/jobs/segment_identify_job_spec.rb

```diff
@@ -65,7 +65,7 @@
         stub_const('ENV', 'LAGO_DISABLE_SEGMENT' => 'true')
 
         expect(SEGMENT_CLIENT).not_to receive(:identify)
-        subject.perform_now(membership_id: membership_id)
+        subject.perform_now(membership_id:)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)